bundle/bin/rke2-uninstall.sh

```diff
@@ -30,13 +34,21 @@ uninstall_disable_services()
 
 uninstall_remove_files()
 {
+
     find "${INSTALL_RKE2_ROOT}/lib/systemd/system" -name rke2-*.service -type f -delete
     rm -f "${INSTALL_RKE2_ROOT}/bin/rke2"
     rm -f "${INSTALL_RKE2_ROOT}/bin/rke2-killall.sh"
     rm -rf "${INSTALL_RKE2_ROOT}/share/rke2"
     rm -rf /etc/rancher/rke2
     rm -rf /var/lib/kubelet
     rm -rf /var/lib/rancher/rke2
+
+    if [ -f "/etc/redhat-release" ]; then
+        rm -f /etc/yum.repos.d/rancher-rke2.repo
+        rm -f /etc/sysconfig/rke2-server
+
+        yum remove rke2-server
```

Comment:
We should probably make this a `yum remove -y rke2-*` as we have RPMs for both `rke2-server`, `rke2-agent`, and also coincidentally have the `rke2-selinux` floating around.

Reply:
and `rke2-common`